src/main/java/stirling/software/SPDF/config/security/database/DatabaseConfig.java

```diff
@@ -0,0 +1,135 @@
+package stirling.software.SPDF.config.security.database;
+
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import stirling.software.SPDF.model.ApplicationProperties;
+import stirling.software.SPDF.model.provider.UnsupportedProviderException;
+
+@Slf4j
+@Getter
+@Configuration
+public class DatabaseConfig {
+
+    public static final String DATASOURCE_DEFAULT_URL =
+            "jdbc:h2:file:./configs/stirling-pdf-DB-2.3.232;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE";
+    public static final String DATASOURCE_URL_TEMPLATE = "jdbc:%s://%s:%4d/%s";
+    public static final String DEFAULT_DRIVER = "org.h2.Driver";
+    public static final String DEFAULT_USERNAME = "sa";
+    public static final String POSTGRES_DRIVER = "org.postgresql.Driver";
+
+    private final ApplicationProperties applicationProperties;
+    private final boolean runningEE;
+
+    @Autowired
+    public DatabaseConfig(ApplicationProperties applicationProperties, boolean runningEE) {
+        this.applicationProperties = applicationProperties;
+        this.runningEE = true; // fixMe: change back
```

Comment:
// fixMe: change back

also autowired can be removed here, its auto added by spring